src/main/scala/viper/gobra/ast/internal/Program.scala

```diff
@@ -79,6 +79,7 @@ sealed trait MethodMember extends MethodLikeMember {
   def results: Vector[Parameter.Out]
   def pres: Vector[Assertion]
   def posts: Vector[Assertion]
+ def terminationMeasure: Option[Vector[Assertion]]
```

Comment:
Please fix the indentation here to match the indentation of the previous methods.